controllers/review.js

```diff
@@ -1,7 +1,13 @@
 const reviewService = require('~/services/review')
 
 const getReviews = async (req, res) => {
-  const reviews = await reviewService.getReviews()
+  let setTargetUserId = {}
+
```

Comment:
const { id } = req.params

const setTargetUserId = id ? { targetUserId: id } : {}

Reply:
Also, you can rename setTargetUserId to TargetUserId or userId

Reply:
just from my point of view, it is not a function that we can call and it will change userId, it is a variable that equal object with id property or empty project

Reply:
FINE! >:0